src/site/content/en/blog/progressively-enhance-your-pwa/index.md

```diff
@@ -0,0 +1,1059 @@
+---
+layout: post
+title: Progressively enhance your Progressive Web App
+subhead: Building for modern browsers and progressively enhancing like it's 2003
+authors:
+  - thomassteiner
+description: |
+  Learn how to progressively enhance your Progressive Web App so that it remains useful
+  on all modern browsers, but delivers an advanced experience on browsers that support
+  new web capabilities like native file system access, system clipboard access,
+  contacts retrieval, periodic background sync, screen wake lock, web sharing features,
+  and many more.
+date: 2020-06-12
+updated: 2020-06-15
```

Comment:
```suggestion
scheduled: true
date: 2020-07-01
# updated: 2020-06-15
```
@tomayac let's just merge this and rely on the fancy `scheduled: true` feature to deploy this at 7:00AM (PST) on the day of your talk?

https://web.dev/handbook/markup-post-codelab/#set-up-the-yaml